components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1665,9 +1559,9 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
 
           qv(i,k) = qv(i,k) + (-qidep+qisub-qinuc)*dt
 
-          th(i,k) = th(i,k) + th(i,k)/t(i,k)*((qidep-qisub+qinuc)*     &
+          th(i,k) = th(i,k) + exner(i,k)*((qidep-qisub+qinuc)*     &
                xxls(i,k)*inv_cp +(qrcol+qccol+qchetc+   &
-               qcheti+qrhetc+qrheti-qimlt)*       &
+               qcheti+qrhetc+qrheti-qimlt)*       &  !AaronDonahue - Is qrmul missing here?
```

Comment:
My knowledge of rime-splintering is small but this does seem problematic if it is the case that splintering does involve a phase change (which I think it should). Could the release of the latent heat fusion associated with the phase be accounted for in one of the other source terms, if so we should check carefully that we are not double accounting for the water source/sinks associate with splintering.

Reply:
Yes, hence my comment.  Having qrmul not be listed here made me uncomfortable. It is a source/sink for ice/rain respectively so you would think it should be included in the temperature tendency.  That being said, qrmul is never defined in the actual code and thus is 0, so it doesn't impact anything.  I would feel better including qrmul here but I wanted to get input from someone with more familiarity with microphysics to make sure I'm not missing something.  @PeterCaldwell , do you have an opinion?

Reply:
I agree that qrmul should affect the temperature. I'm uncomfortable with it being modified (on line 1590) without ever having been set explicitly to zero, and also with how qcmul is declared but never used at all. Presumably qcmul = -qrmul? I suggest we delete all mention of either of these variables. If we decide to add them later, we can start from a clean slate and do it right.

Reply:
Sounds good, I can do that.  Would it be worth determining what (if) other tendency terms are defined and never set/used and remove them as well?  Perhaps including a comment about which tendencies have been given this treatment.

Reply:
Yeah, searching for each defined variable and ensuring it is defined somewhere would probably not take too long...

Reply:
not at all.  I'll go ahead and do that.